### DISCUSSION TITLE
Notification Redirect to Chat screen

### DISCUSSION THREAD

**Discussion by hiashutoshsingh** [GetStream/stream-chat-flutter#393] [Q&A]
Hey, I have set up the notification in my app with stream SDK, I am clueless here how will we redirecting to a specific chat screen since all UI part is catered from SDK side so there is no way to navigate usual way.

Any thoughts?

**Comment by d3xvn** [comment]
Hey @hiashutoshsingh,

We supply components in the SDK rather than whole screens. 
Can you provide some more details over what is inaccessible to you?

In terms of navigation, I'm assuming you're navigating in onLaunch or onResume right? 
(Something like [this](https://medium.com/flutter-community/push-notifications-in-flutter-using-firebase-c1ebaf45d62f))

Thanks.
